packages/drivers/odsp-driver/src/createFile.ts

```diff
@@ -115,14 +116,19 @@ export async function createNewEmptyFluidFile(
                 const { url, headers } = getUrlAndHeadersWithAuth(initialUrl, storageToken);
                 headers["Content-Type"] = "application/json";
 
-                const fetchResponse = await epochTracker.fetchAndParseAsJSON<ICreateFileResponse>(
-                    url,
-                    {
-                        body: undefined,
-                        headers,
-                        method: "PUT",
-                    },
-                    "createFile");
+                const fetchResponse = await runWithRetry(
```

Comment:
Please rename runWithRetry to include at least some hints in the name why retry is needed / type of retry it does
runWithRetryForServiceRecovery or something like that